.gitignore

```diff
@@ -80,3 +80,5 @@ temp/
 **/deployment/global-s3-assets
 **/deployment/regional-s3-assets
 **/deployment/open-source
+docs/excel_import/~$sample.xlsx
```

Comment:
- Aren't we including the `sample.xlsx` file with the solution (adding it to the GitHub repo) for the customers to know how to use the excel import feature?, why adding it to the `.gitignore` file?
- If you want to exclude the file, there is a typo: it should be  `docs/excel_import/sample.xlsx`, not `docs/excel_import/~$sample.xlsx`.

Reply:
docs/excel_import/~$sample.xlsx is a temporary file that Excel creates.